tests/test_api.py

```diff
@@ -530,6 +561,38 @@ def test_assignment_grade(self, mocked_graded_subsections, mocked_get_subsection
             assert learner_data_row['original_grade-lab_ques'] == '3'
             assert learner_data_row['previous_override-lab_ques'] == '5'
 
+    @patch('lms.djangoapps.grades.api.graded_subsections_for_course_id')
+    def test_filter_override_history_columns(self, mocked_graded_subsections):
```

Comment:
could you toss in one for the  "zero" case, where the grades are completely unmodified and we return no subsections?

Reply:
See `test_filter_override_history_noop`